ddsrouter_core/test/blackbox/ddsrouter_core/dds/local/DDSTestLocal.cpp

```diff
@@ -44,7 +44,8 @@ constexpr const uint32_t DEFAULT_MESSAGE_SIZE = 1; // x50 bytes
  * @return configuration::DDSRouterConfiguration
  */
 configuration::DDSRouterConfiguration dds_test_simple_configuration(
-        bool only_builtin_topics = false)
+        bool only_builtin_topics = false,
```

Comment:
Please, refactor this argument name. For me, it means that only builtin topics will be available, but the real meaning is only builtin topics are used and not allowlist ¿?
Maybe `use_dynamic_discovery` is a better name

Reply:
Sorry, but this does still not make sense.  
Why using `use_dynamic_discovery` set builtin topics?

Reply:
I did it this way to not change the original behavior of the test